ios/Plugin/Plugin.swift

```diff
@@ -22,11 +22,27 @@ public class SentryCapacitor: CAPPlugin {
         capOptions.resolve(["value": true]);
     }
     
-    @objc public func setLogLevel() {}
-    
-    @objc private func logLevel() {}
-    
-    @objc public func setUser() {}
+    @objc public func setUser(call: CAPPluginCall) {
+        SentrySDK.configureScope { (scope) in
+            if (call.options["user"] != nil && call.options["otherUserKeys"] != nil) {
```

Comment:
I think this should be `==` not `!=` as you're setting the user to be `nil` if both options are `nil` correct?